index.html

```diff
@@ -26,6 +26,223 @@ <h1>Hey There. I’m Alex</h1>
+                    <button class="big-button" type="button">See Project</button>
+                </div>
+
+                <div class="grid-item">
+                    <div class="multi-post2">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="button">
+                            <div class="flex-container">
+                                <div><button type="button" onclick="#">html</button></div>
+                                <div><button type="button">Bootstrap</button></div> 
+                                <div><button type="button">Ruby</button></div>  
+                              </div>
+                        </div>
+                        
+                    </div>
+                    <button class="big-button" type="button">See Project</button>
+                </div>
+
+                <div class="grid-item">
+                    <div class="multi-post2">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="button">
+                            <div class="flex-container">
+                                <div><button type="button">html</button></div>
+                                <div><button type="button">Bootstrap</button></div> 
+                                <div><button type="button">Ruby</button></div>  
+                              </div>
+                        </div>
+                        
+                    </div>
+                    <button class="big-button" type="button">See Project</button>
+                </div>  
+
+                <div class="grid-item">
+                    <div class="multi-post2">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="button">
+                            <div class="flex-container">
+                                <div><button type="button">html</button></div>
+                                <div><button type="button">Bootstrap</button></div> 
+                                <div><button type="button">Ruby</button></div>  
+                              </div>
+                        </div>
+                        
+                    </div>
+                    <button class="big-button" type="button">See Project</button>
+                </div>
+
+                <div class="grid-item about">
```

Comment:
- [ ]  Great job wrapping your whole headline section. However, note that this would be semantically correct if you wrap it inside a `<section>` tag instead also.